### DISCUSSION TITLE
El bot updatechannel no está funcionando para actualizar properties del canal

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#482] [4. Bug Reports]
Hola a todos

Me di cuenta que el update channel no está actualizando la property del canal de una tarea ($VALUE#answer|channel).
Viendo en los automatic log si indica que lo arregla, pero al revisar en el canal no se visualiza en el campo "elemento". Antes funcionaba.

Dejo la rutina
``
{
  "create_task2": {
    "name": "PBCreateTask",
    "version": "2.0.1",
    "data": {
      "name": [
        "Reunión mejores practicas 14-10-2021"
      ],
      "taskGroup": "5e0f9ce53c2e657fc733173c",
      "user": "59d4f6c1480beb936871c2c2",
      "estimatedTime": "0.5",
      "startDate": "1634212800000",
      "endDate": "1634212800000",
      "parent": "60ddf2fd80bb7343d8dd1497",
      "answers": [
        "616d7a45f6c171c2c22f7f47"
      ],
      "status1": "5e260016717f730c44d600c7",
      "status3": "60ddf2fe1ed9a7000968e8e6",
      "status4": "60257118108da7000871ff78",
      "status5": "6101d348b65e7200072ff91b",
      "editors": [
        "5ccb0d4decf6eb5f34614a97"
      ],
      "assignee": "59d4f6c1480beb936871c2c2",
      "createdBy": "59d4f6c1480beb936871c2c2"
    },
    "next": {
      "SUCCESS": "send_ack",
      "ERROR": ""
    }
  },
  "send_ack": {
    "name": "PBMessage",
    "version": "2.0.0",
    "data": {
      "content": "**Enhorabuena, la tarea N°** 5583 :  Reunión mejores practicas 14-10-2021  **ha sido asignada**",
      "contentType": "text/system",
      "sentBy": "58fe83eda8b7259625b9b17b",
      "channelIds": [
        "60ddf2fe1ed9a7000968e8eb"
      ]
    },
    "next": {
      "SUCCESS": "update_channel",
      "ERROR": ""
    }
  },
  "get_parent": {
    "name": "NWRequest",
    "version": "2.0.0",
    "data": {
      "url": "https://www.cotalker.com/api/tasks/5e0f9ce53c2e657fc733173c/task/60ddf2fd80bb7343d8dd1497",
      "method": "GET",
      "headers": {
        "admin": true
      },
      "defaultAuth": true
    },
    "next": {
      "SUCCESS": "create_task2",
      "ERROR": ""
    }
  },
  "update_channel": {
    "name": "PBUpdateChannel",
    "version": "2.0.0",
    "data": {
      "channelId": "616d7a5b20bc8d0007aeec8b",
      "propertyIds": [
        "6101d348b65e7200072ff91b"
      ],
      "isActive": true
    },
    "next": {
      "SUCCESS": "",
      "ERROR": ""
    }
  }
}
``

Dejo el log
``
[
  {
    "key": "get_parent",
    "name": "NWRequest",
    "status": "SUCCESS",
    "result": {
      "statusCode": 200,
      "data": {
        "_id": "60ddf2fd80bb7343d8dd1497",
        "public": {
          "isActive": false,
          "language": "english",
          "title": "",
          "response": ""
        },
        "answers": [
          "60ddc75af6c171c2c242089a"
        ],
        "indentation": 0,
        "isActive": true,
        "isValid": false,
        "activeSlas": [],
        "validators": [],
        "editors": [
          "59d4f6c1480beb936871c2c2",
          "5a5533f1407011dc14d1bbff",
          "5ccb0d4decf6eb5f34614a97",
          "5ce58ed8cca875a52d8cda18",
          "5e381966c1346f6eb75cdc02",
          "60c0e9d93ca668000867b75d",
          "612f9710d8db3f0007b8acb1",
          "5de67182b5add43684b4ba97",
          "5fa95476f617b300080254b3"
        ],
        "followers": [],
        "visibility": [],
        "userList": [
          "59d4f6c1480beb936871c2c2",
          "5a5533f1407011dc14d1bbff",
          "5ccb0d4decf6eb5f34614a97",
          "5ce58ed8cca875a52d8cda18",
          "5e381966c1346f6eb75cdc02",
          "60c0e9d93ca668000867b75d",
          "612f9710d8db3f0007b8acb1",
          "5de67182b5add43684b4ba97",
          "5fa95476f617b300080254b3"
        ],
        "color": "none",
        "formMessage": [],
        "language": "english",
        "name": " COTALKER  -  2021-2",
        "relativeWeight": 1,
        "assignee": "59d4f6c1480beb936871c2c2",
        "status1": "5e260016717f730c44d600c7",
        "createdBy": "59d4f6c1480beb936871c2c2",
        "company": "58fe83eda8b7259625b9b15d",
        "taskGroup": "5e0f9ce53c2e657fc733173c",
        "owner": "manual",
        "weight": 0,
        "projectCode": "60ddf2fe1ed9a7000968e8ec",
        "createdAt": "2021-07-01T16:53:17.903Z",
        "status": "5e0f9770ace7ee7f3895d238",
        "smState": "5e0fb5c9ed59007f744bb542",
        "smStateMachine": "5e0fb59bace7ee7f38971187",
        "modifiedStateAt": "2021-09-06T14:21:39.923Z",
        "closedAt": null,
        "asset": "60ddf2fe1ed9a7000968e8e6",
        "channel": "60ddf2fe1ed9a7000968e8eb",
        "serial": 4516,
        "modifiedAt": "2021-09-29T17:12:40.036Z",
        "__v": 5,
        "endDate": null,
        "estimatedTime": null,
        "info": null,
        "startDate": null,
        "status2": null,
        "status3": null,
        "status4": null,
        "status5": null,
        "image": {
          "small": "https://cotalker-us-files.s3.amazonaws.com/cotalkerreframe/image/v4_610424733a9f8dfad7dece98/small/administrador-16-06-2020.png",
          "square": "https://cotalker-us-files.s3.amazonaws.com/cotalkerreframe/image/v4_610424733a9f8dfad7dece98/square/administrador-16-06-2020.png",
          "original": "https://cotalker-us-files.s3.amazonaws.com/cotalkerreframe/image/v4_610424733a9f8dfad7dece98/original/administrador-16-06-2020.png"
        },
        "extensions": {
          "project_assets": {
            "horas_proyecto": null,
            "valor_hora_proyecto": null,
            "fecha_produccion": "",
            "fecha_kickoff": "",
            "numero_usuarios": null,
            "valor_saas": null,
            "valor_setup": ""
          }
        }
      }
    },
    "nextBot": "create_task2",
    "iteration": 1
  },
  {
    "key": "create_task2",
    "name": "PBCreateTask",
    "status": "SUCCESS",
    "result": {
      "task": {
        "public": {
          "isActive": false,
          "language": "english",
          "title": "",
          "response": ""
        },
        "answers": [
          "616d7a45f6c171c2c22f7f47"
        ],
        "indentation": 1,
        "isActive": true,
        "isValid": false,
        "activeSlas": [],
        "validators": [],
        "editors": [
          "5ccb0d4decf6eb5f34614a97",
          "59d4f6c1480beb936871c2c2",
          "5a5533f1407011dc14d1bbff",
          "5ce58ed8cca875a52d8cda18",
          "5e381966c1346f6eb75cdc02",
          "60c0e9d93ca668000867b75d",
          "612f9710d8db3f0007b8acb1",
          "5de67182b5add43684b4ba97",
          "5fa95476f617b300080254b3"
        ],
        "followers": [],
        "visibility": [],
        "userList": [
          "5ccb0d4decf6eb5f34614a97",
          "59d4f6c1480beb936871c2c2",
          "5a5533f1407011dc14d1bbff",
          "5ce58ed8cca875a52d8cda18",
          "5e381966c1346f6eb75cdc02",
          "60c0e9d93ca668000867b75d",
          "612f9710d8db3f0007b8acb1",
          "5de67182b5add43684b4ba97",
          "5fa95476f617b300080254b3"
        ],
        "color": "none",
        "formMessage": [],
        "language": "english",
        "_id": "616d7a5c59228143748426d9",
        "name": " Reunión mejores practicas 14-10-2021",
        "relativeWeight": 1,
        "assignee": "59d4f6c1480beb936871c2c2",
        "parent": "60ddf2fd80bb7343d8dd1497",
        "estimatedTime": 0.5,
        "startDate": "2021-10-14T12:00:00.000Z",
        "endDate": "2021-10-14T12:00:00.000Z",
        "status1": "5e260016717f730c44d600c7",
        "status3": "60ddf2fe1ed9a7000968e8e6",
        "status4": "60257118108da7000871ff78",
        "status5": "6101d348b65e7200072ff91b",
        "createdBy": "59d4f6c1480beb936871c2c2",
        "company": "58fe83eda8b7259625b9b15d",
        "taskGroup": "5e0f9ce53c2e657fc733173c",
        "owner": "manual",
        "weight": 1099511627776,
        "projectCode": "60ddf2fe1ed9a7000968e8ec",
        "createdAt": "2021-10-18T13:45:00.477Z",
        "status": "5e18b152416aa50c6b3fd104",
        "smState": "5e18d058416aa50c6b415142",
        "smStateMachine": "5e18d034717f730c449ac56b",
        "modifiedStateAt": "2021-10-18T13:45:00.549Z",
        "closedAt": null,
        "asset": "5d31ed203e96693bc79bc004",
        "channel": "616d7a5c4d218a0007b8bb92",
        "serial": 5583,
        "modifiedAt": "2021-10-18T13:45:00.645Z",
        "__v": 0,
        "extensions": {}
      }
    },
    "nextBot": "send_ack",
    "iteration": 2
  },
  {
    "key": "send_ack",
    "name": "PBMessage",
    "status": "SUCCESS",
    "result": {
      "status": true
    },
    "nextBot": "update_channel",
    "iteration": 3
  },
  {
    "key": "update_channel",
    "name": "PBUpdateChannel",
    "status": "SUCCESS",
    "result": {
      "status": true,
      "channel": {
        "settings": {
          "display": "normal",
          "read": "all",
          "write": "all",
          "readUserIds": [],
          "writeUserIds": []
        },
        "visibility": {
          "all": false,
          "groupRoles": [],
          "users": []
        },
        "info": {
          "users": [],
          "properties": [],
          "action": [],
          "startDate": "2021-10-18T13:44:59.785Z"
        },
        "lastMessage": {
          "content": "Nicolás Peña y Lillo Goya: Has sent a form",
          "date": "2021-10-18T13:44:59.841Z",
          "sender": "59d4f6c1480beb936871c2c2",
          "contentType": "application/vnd.cotalker.survey"
        },
        "videoCall": {
          "start": {
            "any": false,
            "permissions": []
          },
          "isActive": false,
          "duration": [],
          "publishVideo": [],
          "publishAudio": []
        },
        "propertyIds": [
          "6101d348b65e7200072ff91b"
        ],
        "userIds": [
          "59d4f6c1480beb936871c2c2"
        ],
        "archive": [],
        "groupOwnerIds": [
          "59d4f6c1480beb936871c2c2"
        ],
        "isPrivate": true,
        "isDirect": false,
        "isActive": true,
        "pinned": [],
        "bots": [],
        "search": [
          "pre",
          "bound",
          "1634564699784",
          "5e18baf3229bf30c57922004",
          "60ddf2fd80bb7343d8dd1497",
          "gp",
          "crear",
          "tarea",
          "al",
          "proyecto",
          "2021",
          "10",
          "18",
          "prebound16345646997845e18baf3229bf30c5792200460ddf2fd80bb7343d8dd1497",
          "20211018"
        ],
        "_id": "616d7a5b20bc8d0007aeec8b",
        "company": "58fe83eda8b7259625b9b15d",
        "group": "5e0f9ce5bd891701378ba6d4",
        "groupNumber": 37,
        "overrideName": true,
        "nameDisplay": "(GP) Crear tarea al proyecto - 2021-10-18",
        "nameCode": "pre-bound#1634564699784#5e18baf3229bf30c57922004#60ddf2fd80bb7343d8dd1497",
        "lightProperty": [],
        "createdAt": "2021-10-18T13:44:59.785Z",
        "modifiedAt": "2021-10-18T13:45:00.976Z",
        "autocomplete": [],
        "actionButton": [],
        "__v": 0
      }
    },
    "nextBot": "",
    "iteration": 4
  }
]
``

gracias =>

**Comment by jdurandpaz** [comment]
@NicolasPLG por favor indicar si pudiste probar con el update task
